src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -3004,8 +3004,8 @@ private StatementAst ConfigurationStatementRule(IEnumerable<AttributeAst> custom
                         dropIntoDebugger: true);
                 }
 
-                // Configuration is not supported on ARM64
-                if (PsUtils.IsRunningOnProcessorArchitectureARM())
+                // Configuration is not supported for ARM or ARM64 process architecture.
+                if (PsUtils.IsRunningOnProcessArchitectureARM())
```

Comment:
@SteveL-MSFT You only mentioned ARM64 not supported in https://github.com/PowerShell/PowerShell/issues/18781#issuecomment-1419786229, not ARM, so should we only check for ARM64 here?

Reply:
The `configuration` keyword is only useful on Windows and Windows only supports Arm64 (no Arm32).  However, for the code, it's ok to check for ARM as long as it's a superset of 32 and 64.  There should be no functional difference as we don't produce Windows Arm32 packages anymore.

Reply:
So, we are good with the current check.